Crash when enable minify

```
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.numero.material_gallery/com.numero.material_gallery.MainActivity}: android.view.InflateException: Binary XML file line #40 in com.numero.material_gallery:layout/activity_main: Binary XML file line #40 in com.numero.material_gallery:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3633)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3790)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2202)
        at android.os.Handler.dispatchMessage(Handler.java:106)
        at android.os.Looper.loopOnce(Looper.java:201)
        at android.os.Looper.loop(Looper.java:288)
        at android.app.ActivityThread.main(ActivityThread.java:7829)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:982)
     Caused by: android.view.InflateException: Binary XML file line #40 in com.numero.material_gallery:layout/activity_main: Binary XML file line #40 in com.numero.material_gallery:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
     Caused by: android.view.InflateException: Binary XML file line #40 in com.numero.material_gallery:layout/activity_main: Error inflating class androidx.fragment.app.FragmentContainerView
     Caused by: java.lang.IllegalArgumentException: No @Navigator.Name annotation found for z
        at c1.z0.c(Unknown Source:49)
        at c1.z0.a(Unknown Source:117)
        at c1.m.<init>(Unknown Source:169)
        at c1.a0.<init>(Unknown Source:53)
        at androidx.navigation.fragment.NavHostFragment.n0(Unknown Source:219)
        at x0.f0.S0(Unknown Source:2949)
        at x0.v1.e(Unknown Source:475)
        at x0.v1.m(Unknown Source:278)
        at x0.l1.V(Unknown Source:2189)
        at x0.l1.M0(Unknown Source:2106)
        at x0.l1.T(Unknown Source:1971)
        at x0.a.k(Unknown Source:311)
        at androidx.fragment.app.FragmentContainerView.<init>(Unknown Source:180)
        at x0.s0.onCreateView(Unknown Source:52)
        at x0.o0.v(Unknown Source:135)
        at x0.j0.s(Unknown Source:319)
        at x0.j0.onCreateView(Unknown Source:298)
        at android.view.LayoutInflater.tryCreateView(LayoutInflater.java:1073)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:1001)
        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:965)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1127)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1088)
        at android.view.LayoutInflater.rInflate(LayoutInflater.java:1130)
        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:1088)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:686)
        at android.view.LayoutInflater.inflate(LayoutInflater.java:538)
        at k5.a.d(Unknown Source:65)
        at k5.a.c(Unknown Source:59)
        at com.numero.material_gallery.MainActivity.onCreate(Unknown Source:56)
        at android.app.Activity.performCreate(Activity.java:8051)
        at android.app.Activity.performCreate(Activity.java:8031)
        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1329)
        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3606)
        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3790)
        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:103)
        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
```

--- a/build.gradle
+++ b/build.gradle
@@ -8,7 +8,7 @@ buildscript {
 
                 kotlin    : "1.5.21",
 
-                navigation: "2.3.5",
+                navigation: "2.4.0-alpha06",
                 lifecycle : "2.4.0-alpha02",
 
                 hilt      : "2.38.1",
@@ -24,7 +24,7 @@ buildscript {
                 androidx    : [
                         appcompat       : "androidx.appcompat:appcompat:1.3.0-rc01",
                         activity        : "androidx.activity:activity-ktx:1.3.1",
-                        fragment        : "androidx.fragment:fragment-ktx:1.3.6",
+                        fragment        : "androidx.fragment:fragment-ktx:1.4.0-alpha06",
                         core            : "androidx.core:core-ktx:1.6.0-beta02",
                         recyclerview    : "androidx.recyclerview:recyclerview:1.2.1",
                         constraintlayout: "androidx.constraintlayout:constraintlayout:2.0.4",

